processor/src/main/java/org/mapstruct/ap/internal/model/ValueMappingMethod.java

```diff
@@ -117,13 +118,17 @@ else if ( sourceType.isString() && targetType.isEnumType() ) {
                 LifecycleMethodResolver.beforeMappingMethods( method, selectionParameters, ctx, existingVariables );
             List<LifecycleCallbackMethodReference> afterMappingMethods =
                 LifecycleMethodResolver.afterMappingMethods( method, selectionParameters, ctx, existingVariables );
-            AdditionalAnnotationsBuilder additionalAnnotationsBuilder =
+            List<Annotation> annotations = Collections.emptyList();
```

Comment:
you initialize the list of annotations twice.. Why not write:
```java
List<Annotations> annotations;
if ( method instanceof ForgedMethod ) {
   annotations = Collections.emptyList();
}
else {
  annotations = new ArrayList();
  // ...
}
```

Reply:
@sjaakd Thank you for the review, I just don't like writing else, I will try to modify it :)